lib/DidDocumentKey.ts

```diff
@@ -0,0 +1,41 @@
+import Encoder from './Encoder';
+import ErrorCode from './ErrorCode';
+import IonError from './IonError';
+import PublicKeyPurpose from './models/PublicKeyPurpose';
+
+/**
+ * Class containing operations related to keys specified within a DID Document.
+ */
+export default class DidDocumentKey {
```

Comment:
Looking for feedback on better consolidation of naming of `DidDocumentKey`, `PublicKeyModel`, and `PublicKeyPurpose`.

Reply:
DidDocumentKey makes sense to me

Reply:
Discussed renames offline.